derive/src/lib.rs

```diff
@@ -123,6 +123,38 @@
 //!
 //! # fn main() {}
 //! ```
+//! 
+//! Client Example
+//! 
+//! ```
+//! use jsonrpc_derive::rpc;
+//! use jsonrpc_core::futures::future::Future;
+//! use jsonrpc_core::futures::sync::mpsc;
+//! 
+//! #[rpc(client)]
+//! pub trait Rpc {
+//! 	//! / Returns a protocol version
+//! 	#[rpc(name = "protocolVersion")]
+//! 	fn protocol_version(&self) -> Result<String>;
+//! 
+//! 	//! / Adds two numbers and returns a result
+//! 	#[rpc(name = "add", alias("callAsyncMetaAlias"))]
+//! 	fn add(&self, a: u64, b: u64) -> Result<u64>;
+//! }
+//! 
+//! fn main() {
+//! 	let fut = {
+//! 		let (sender, _) = mpsc::channel(0);
```

Comment:
I'd add a comment what the receiver is for (or even spawn a new thread that actually responds).

Reply:
I've updated it to use the client/server example (copied from `derive/examples/std`)